### PR TITLE
Pull cpu usage predicition model from S3

### PR DESCRIPTION
**Pull request by gabrielhartmann**

### CODE REVIEW

File: titus_isolate/allocate/utils.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+from titus_isolate.config.utils import get_required_property
+from titus_isolate.utils import get_config_manager
+
+
+def get_cpu_model_bucket_name():
+    format_str = get_required_property(MODEL_BUCKET_FORMAT_STR)
+    if format_str is None:
+        return None
+
+    region = os.environ['EC2_REGION']
```
Comment (sargun) [inline titus_isolate/allocate/utils.py]: Any reason why you don't just "prerender" the bucket name as an env variable? And then leave it to some config file to figure this out?

File: titus_isolate/allocate/utils.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+    format_str = get_config_manager().get(MODEL_PREFIX_FORMAT_STR)
+    if format_str is None:
+        return None
+
+    return format_str.format(prefix, leaf)
+
+
+def get_cpu_models():
+
```
Comment (sargun) [inline titus_isolate/allocate/utils.py]: Extra newline?

File: titus_isolate/allocate/utils.py (hunk 1 above)
Comment (gabrielhartmann) [reply titus_isolate/allocate/utils.py]: The current method is based on two priorities.
1. Don't expose actual bucket names and paths in the repository
2. Be able to redirect titus-isolate to download a model from an alternative location in an emergency

If bucket name is pre-rendered as part of the deb / systemd definition we leak some information.

File: titus_isolate/allocate/utils.py (hunk 2 above)
Comment (gabrielhartmann) [reply titus_isolate/allocate/utils.py]: Thanks.  Fixed.

File: titus_isolate/allocate/utils.py (hunk 1 above)
Comment (sargun) [reply titus_isolate/allocate/utils.py]: If you look at Titus-executor the way it's handled is by doing this:
https://github.com/Netflix/titus-executor/blob/master/root/apps/titus-executor/bin/run#L17-L22

And then can construct variables dynamically.

Comment (gabrielhartmann) [reply titus_isolate/allocate/utils.py]: I don't understand the benefit and this would appear to violate (1).

**Pull request merged**